terragrunt/bootstrap/module/scripts/firewallinit.yml

```diff
@@ -49,16 +49,26 @@ write_files:
   - path: /etc/ssh/sshd_config.d/55-password-yes.conf
     content: |
       PasswordAuthentication yes
+
+  - path: /etc/hosts
+    content: |
+      # Added by cloud-config
+      192.168.50.254 inetfw
+    owner: root:root
+    permissions: '0644'
  
 bootcmd:
   - echo "nameserver ${dns_server_address}" > /etc/resolv.conf
+  - ip route del default via 192.42.0.1 dev ens3 proto dhcp src 192.42.0.254 metric 100
```

Comment:
can't we manage that somehow with netplan?

Reply:
done

Reply:
@whotwagner nevermind, dhcp overrides that again, i can disable dhcp in netplan for ens4 through ens7 and configure ips manually. if thats what we want?